src/main/java/ita/softserve/course_evaluation/dto/UserDtoMapper.java

```diff
@@ -0,0 +1,36 @@
+package ita.softserve.course_evaluation.dto;
+
+import ita.softserve.course_evaluation.entity.User;
+import org.modelmapper.ModelMapper;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+
+@Component
+public class UserDtoMapper {
+
+        private static ModelMapper mapper;
+
+    public static UserDto toDto(User user) {
```

Comment:
reformat code

Reply:
I have rewritten the one without ModelMapper